trade-finance-manager-api/src/v1/controllers/amendment.controller.js

```diff
@@ -227,7 +245,16 @@ const updateFacilityAmendment = async (req, res) => {
       if (payload?.taskUpdate?.updateTask) {
         const tasks = await updateAmendmentTasks(facilityId, amendmentId, payload.taskUpdate);
         payload.tasks = tasks;
-        payload.ukefDecision = { isReadyForApproval: isRiskAnalysisCompleted(tasks) };
+
+        /**
+         * Only write amendment UKEF decision if not
+         * already written by underwriters, otherwise
+         * UW's decision will be removed.
+         */
+        if (!amendment.ukefDecision) {
```

Comment:
Do we have unit tests for this being true and false? I can't see

Reply:
Have added.